api/server/volume.go

```diff
@@ -1847,6 +1884,7 @@ func (vd *volAPI) otherVolumeRoutes() []*Route {
 		{verb: "POST", path: volPath("/unquiesce/{id}", volume.APIVersion), fn: vd.unquiesce},
 		{verb: "GET", path: volPath("/catalog/{id}", volume.APIVersion), fn: vd.catalog},
 		{verb: "POST", path: volPath("/volservice/{id}", volume.APIVersion), fn: vd.VolService},
+		{verb: "GET", path: volPath("/util/{id}", volume.APIVersion), fn: vd.volumeBytesUsedByNode},
```

Comment:
Should the path be /consumed/{id} or /bytes_used as Utils might not relay the functionality

Reply:
ok can revise to bytes_used